webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2961,7 +2971,11 @@ var userOverride = {
                     'aria-label',
                     LEAD_COLLECTION_LABEL.submit
                 );
-                leadCollectionHeading.innerHTML = LEAD_COLLECTION_LABEL.heading;
+                if(appOptions.preLeadCollection){
+                    leadCollectionHeading.innerHTML= LEAD_COLLECTION_LABEL.heading;
+                }else{
+                    leadCollectionHeading.innerHTML= appOptions.appSettings.chatWidget.preChatGreetingMsg;   
```

Comment:
what will happen if pre chat greeting message is not set?

Reply:
made greeting message required in the dashboard section.

Reply:
why? what if a customer doesn't want the greeting message?

Reply:
in that case, do I have to remove the div from the chat widget which contains the greeting message title?

Reply:
> in that case, do I have to remove the div from the chat widget which contains the greeting message title?

![image](https://user-images.githubusercontent.com/22856546/108737014-2b5cb580-7558-11eb-9333-539c90a9ae70.png)

Reply:
sorry this is fine, i got confused with the other greeting message